src/components/Tags/TagBadge.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+
+import {
+  TagConfig,
+  getTagHierarchyDisplay,
+} from "@/types/emr/tagConfig/tagConfig";
+
+import { Badge } from "@/components/ui/badge";
+import { cn } from "@/lib/utils";
+
+function getTagColorStyles(tag: TagConfig) {
+  const color = tag.meta.color as string;
+  if (!color) return {};
+
+  return {
+    color,
+    backgroundColor: color + "40",
```

Comment:
🤔  what does this mean? can you give an example value for tag.meta.color?

Edit: Nvm, hashcode value, but still this is confusing 😅